### PR TITLE
Fix faulty upgrade step introduced in 2.3.1.

### PR DESCRIPTION
**Pull request by mbaechtold**
The upgrade step introduced in https://github.com/4teamwork/ftw.slider/pull/49 seems to purge the existing behaviors.

### CODE REVIEW

**Comment by mbaechtold** [comment]
@jone, should I create an addition upgrade step, which re-adds the behaviors defined in https://github.com/4teamwork/ftw.slider/blob/78ca9dc2fb77b121ed07aef36b5739241be25472/ftw/slider/profiles/default/types/ftw.slider.Pane.xml#L28 ?

**Comment by jone** [comment]
Yes, probably. I'd specifically only add `INameFromTitle` though.

**Comment by mbaechtold** [comment]
@jone, I have added an upgrade step which adds `plone.app.content.interfaces.INameFromTitle` to the FTI.

I have also added a warning in the changelog:

https://github.com/4teamwork/ftw.slider/blob/4a5f66b084183e8c03a016eff4d6a1f89dded69c/docs/HISTORY.txt#L8-L16

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,16 @@ Changelog
 3.1.3 (unreleased)
 ------------------
 
+- Fix faulty upgrade step introduced in 2.3.1.
+
+  Warning: If your upgrade to this version, the behavior
```
Comment (jone) [inline docs/HISTORY.txt]: `you`, `you're`, but not `your`, I think

File: ftw/slider/upgrades/20170824081802_add_i_name_from_title_behavior_to_ftw_slider_pane/types/ftw.slider.Pane.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object name="ftw.slider.Pane">
+
+    <property name="behaviors" purge="False">
+        <element value="plone.app.content.interfaces.INameFromTitle"/>
```
Comment (jone) [inline ftw/slider/upgrades/20170824081802_add_i_name_from_title_behavior_to_ftw_slider_pane/types/ftw.slider.Pane.xml]: Does this ensure uniqueness? 🤔 

The fix of the old upgrade step makes that the `INameFromTitle` behavior is no longer removed in the old upgrade, but it is still added here. Will it then end up twice in the behaviors list? 🤔

Comment (mbaechtold) [inline ftw/slider/upgrades/20170824081802_add_i_name_from_title_behavior_to_ftw_slider_pane/types/ftw.slider.Pane.xml]: Yes, uniqueness is guaranteed. If you run an upgrade step, which add behavior, multiple times, the behavior is *not* added multiple times.

File: docs/HISTORY.txt (hunk 1 above)
Comment (mbaechtold) [inline docs/HISTORY.txt]: Nice catch...

**Comment by mbaechtold** [comment]
Thank you for the review, @jone!

**Pull request merged**